Source/FieldSolver/FiniteDifferenceSolver/EvolveB.cpp

```diff
@@ -0,0 +1,76 @@
+#include <WarpXAlgorithmSelection.H>
+#include<FiniteDifferenceAlgorithms/YeeAlgorithm.H>
+#include<FiniteDifferenceSolver.H>
+#include<AMReX_Gpu.H>
+
+using namespace amrex;
+
+void FiniteDifferenceSolver::EvolveB ( VectorField& Bfield,
+                                       VectorField& Efield,
```

Comment:
E is probably const then:
```suggestion
                                       VectorField const& Efield,
```